src/FixedDataTable.js

```diff
@@ -556,6 +594,7 @@ var FixedDataTable = createReactClass({
 
       scrollbarYHeight = Math.max(0, footOffsetTop - bodyOffsetTop);
     }
+    this._scrollbarYHeight = scrollbarYHeight;
```

Comment:
Is this related to this change?

Reply:
Sorry, a few other projects came up.  I agree that it needs a flag before being merged.  I plan to add it as soon as I have a moment.  The saving of the computed scrollbarYHeight is part of the change.  It is used in the event handler to figure out how tall "1 page" is for page up/page down.

Reply:
Can we save this to state instead of having it as a property?

Reply:
The code that computes that value is in render.  If memory serves react really doesn't like it when you update state inside of render.  Also it isn't actually part of the UI state, more a computed value that has been cached.  It never gets used to render anything which is why I was ok with using a property.